src/fs/path.ts

```diff
@@ -10,11 +10,17 @@ export const join = (parts: string[]): string => {
 
 export const joinNoSuffix = (parts: string[]): string => {
   const joined = join(parts)
-  return joined[joined.length -1] === '/' 
+  return joined[joined.length -1] === '/'
           ? joined.slice(0, joined.length -1)
           : joined
 }
 
+export const parent = (path: string): string | null => {
```

Comment:
You can also do this with `takeTail`
```ts
return takeTail(path).parentPath
```
but doesn't really matter :shrug: 
I don't love the names `takeTail` and `takeHead` either but couldn't think of better names

Reply:
Ah cool. Will alias this function that then.